graphql-java-servlet/src/main/java/graphql/servlet/SimpleGraphQLHttpServlet.java

```diff
@@ -2,38 +2,66 @@
 
 import graphql.schema.GraphQLSchema;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 /**
  * @author Andrew Potter
  */
 public class SimpleGraphQLHttpServlet extends AbstractGraphQLHttpServlet {
```

Comment:
After refactoring of `AbstractGraphQLHttpServlet` this class can probably be removed as well.